### ISSUE TITLE
Set up Elements Repository Integration Tools on development server

### ISSUE THREAD

**Issue by amandafrench** [VTUL/vtechworks#210]
Waiting for Symplectic to finish migrating data and setting up an Elements development server; also waiting on completion of a metadata crosswalk

**Comment by keithgee** [comment]
The tools are installed. I have some fixes/documenting to do for install scripts, but there is a separate issue open for that. 

We'll open separate issues for anything we find during testing, I think. Good?
**Closing

**Comment by keithgee** [comment]
I should have noted that the connection is from ED -> PPRD.
